moviEATS.js

```diff
@@ -56,13 +54,16 @@ var getMoviesAPI = function(input) {
       if (keys.includes(checkMovie[0])) {
         movieGenre.textContent = movie.Genre;
         output.setAttribute('class', 'show');
```

Comment:
Cool added a for loop to iterate through the recipes to display them x number of times.